libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -315,26 +313,23 @@ - (BOOL)handleAdvancedAuthenticationResponse:(NSURL *)appUrlResponse {
         [self log:SFLogLevelInfo format:@"%@ Current advanced auth state (%@) not compatible with handling app launch auth response.", NSStringFromSelector(_cmd), [[self class] advancedAuthStateDesc:self.advancedAuthState]];
         return NO;
     }
-    
+    [SFSDKAppFeatureMarkers registerAppFeature:kSFAppFeatureSafariBrowserForLogin];
```

Comment:
Do you want this where the advanced browser flow has actually fully completed?  This isn't quite there.